lib/vistle/control/vistle.py

```diff
@@ -296,17 +296,23 @@ def load(filename = None):
 def loadScript(filename):
    _vistle.loadScript(filename)
 
-def moduleCompoundAlloc(compoundName):
-   return _vistle.moduleCompoundAlloc(compoundName)
+def moduleCompoundCreate(compoundName):
+   return _vistle.moduleCompoundCreate(compoundName)
+
+def moduleCompoundSetPath(compoundId, path):
+   _vistle.moduleCompoundSetPath(compoundId, path)
 
 def moduleCompoundAddModule(compoundId, moduleName, x, y):
    return _vistle.moduleCompoundAddModule(compoundId, moduleName, x, y)
 
 def moduleCompoundConnect(compoundId, fromId, fromName, toId, toName): #if fromId or toId is equal to compound id the port is exposed with the given name
    _vistle.moduleCompoundConnect(compoundId, fromId, fromName, toId, toName)
 
-def moduleCompoundCreate(compoundId):
-   _vistle.moduleCompoundCreate(compoundId)
+def moduleCompoundExpose(compoundId, compoundPortName, modId, modPortName, portType):
+   _vistle.moduleCompoundExpose(compoundId, compoundPortName, modId, modPortName, portType)
```

Comment:
how about having three of them?
- moduleCompoundExposeInput
- moduleCompoundExposeOutput
- moduleCompoundExposeParameter